src/System.Runtime.Intrinsics/Directory.Build.props

```diff
@@ -3,8 +3,7 @@
   <Import Project="..\Directory.Build.props" />
   <PropertyGroup>
     <AssemblyVersion>4.0.1.0</AssemblyVersion>
```

Comment:
Should we rev the version?

Reply:
We bumped the version after shipping `2.1` here: https://github.com/dotnet/corefx/commit/6a1b5c0aeee747a535579da7374c05b8d878fb65.  So I don't think we need to rev it.

Reply:
What about UAP? Do we plan to support it there?

Reply:
Also given this is an new assembly identity I would reset the version to 4.0.0.0 as there isn't any reason to keep it at 4.0.1.0.

Reply:
> What about UAP? Do we plan to support it there?

UAP doesn't support these APIs - only netcoreapp.

See https://github.com/dotnet/corefx/blob/d24a86c443735f465e5b15398f1ed4c9eb607b80/src/System.Runtime.Intrinsics.Experimental/src/Configurations.props#L4-L7

Reply:
Are there plans to add UAP support for these?

Reply:
It requires CoreRT work to make them functional (and deciding how to support aot)

Reply:
OK I was just curious. I do still think it is worth resetting the version to 4.0.0.0 in case you missed that comment @tannergooding

Reply:
Reset the version to `4.0.0.0`